### ISSUE TITLE
How to upgrade openmvg to the newest version

### ISSUE THREAD

**Issue by MEGA728** [openMVG/openMVG#892]
Hi ! I would like to upgrade the openmvg on my computer from v1.0 to v1.1, how can I do it? 
Do I have to delete the old libs before using CMake to generate the project files and using VS to build the new libs?

**Comment by pmoulon** [comment]
Yes if you are under windows you can delete the old libs and do cmake, make, and install.
